test/conftest.py

```diff
@@ -46,6 +48,30 @@ def skipif_pre_ansible211(is_pre_ansible211):
         pytest.skip("Valid only on Ansible 2.11+")
 
 
+@pytest.fixture(scope="session")
```

Comment:
I'm not crazy about what the `is_pre_ansible211` test does. It doesn't seem like it'll work in the general case.

Reply:
I'm also not sure how I feel about this implementation generally but it gets us there quick with whatever variant of Ansible is installed in the environment.

Reply:
The `is_pre_ansible211` fixture should work in the general case. The ansible package  used in our tests will be `ansible-core` for 2.11 or higher. 2.10 is `ansible-base`, and 2.9 and earlier is `ansible`.